kahuna/public/js/search/index.js

```diff
@@ -35,15 +35,16 @@ export var search = angular.module('kahuna.search', [
 search.config(['$stateProvider', '$urlMatcherFactoryProvider',
                function($stateProvider, $urlMatcherFactoryProvider) {
 
-    // Query params slashes are encoded that mess up our seach query when we search for e.g. `24/7`.
-    // see: https://github.com/angular-ui/ui-router/issues/1119#issuecomment-64696060
-    // TODO: Fix this
-    $urlMatcherFactoryProvider.type('Slashed', {
-        encode: val => angular.isDefined(val) ? val : undefined,
-        decode: val => angular.isDefined(val) ? val : undefined,
-        // We want to always match this type. If we match on slash, it won't update
-        // the `stateParams`.
-        is: () => true
+    const zeroWidthSpace = '\u200B';
+    function removeUtf8SpecialChars(val) {
+        return val && val.replace(zeroWidthSpace, '');
```

Comment:
This only replaces the first one. Did we want to be safe and replace all zero-width space?